### DISCUSSION TITLE
seqr individual id

### DISCUSSION THREAD

**Discussion by sophienguyen01** [broadinstitute/seqr#4145] [General]
Hi,
I'm using local install via docker, and when I click Edit Datasets, it asks me to load `ID Mapping File Path`. I have no idea what this is. Where do I get the seqr individual ids?
![Screenshot 2024-06-06 at 11 06 38 AM](https://github.com/broadinstitute/seqr/assets/135994439/8e621ad6-47a1-431b-b2c0-35f95f4afc5e)

THank you

**Comment by hanars** [comment]
The sample IDs in that error message are in your loaded data but not in your seqr project. Your options are:
1) You do not want to load data for these samples. In that case, you should select the "Ignore extra samples" checkbox
2) You want to load data for them with these sample IDs. In that case, you need to create the families and individuals before loading
3) You want to load data for them, but the sample IDs in the data do not match the Individual IDs for the individuals in the project. In that case, you upload the mapping file mapping these sample IDs to their corresponding seqr individual IDs

**Comment by sophienguyen01** [comment]
HI,

This is my situation: I loaded the data using instructions on this link: 
![image](https://github.com/broadinstitute/seqr/assets/135994439/1b10e677-10b7-464f-962e-dc8c1ba0ccc1)
Got this command finished successfully: `docker-compose exec pipeline-runner load_data.sh $BUILD_VERSION $SAMPLE_TYPE $INDEX_NAME $INPUT_FILE_PATH`.
Then I go to Seqr website, and I don't see any project listed. I did see my index in `Data management/Elasticsearch Status` though. So I created new project => 'Edit Datasets' and it is where I encountered problems.

With option1, I tried and this is the error:
![Screenshot 2024-06-06 at 11 59 54 AM](https://github.com/broadinstitute/seqr/assets/135994439/643fb5b5-e3a4-4ea5-9284-ceaf58698a84)


For option2 and 3, how do I create the data that you mentioned? Could you guide my to an example? I am trying to find an example on seqr github but could not find anything. 
Also, I have only 1 sample in my input vcf and how does it work?

**Reply by hanars** [reply]
You need to have an Individual in your project to correspond to the sample in your VCF. You do this via the "Edit Families and Individuals" option on the project page. You need to do this prior to data loading

**Reply by sophienguyen01** [reply]
Did you mean that I need to add file to 'Edit Families and Individuals' before running this command on local premise:
`docker-compose exec pipeline-runner load_data.sh $BUILD_VERSION $SAMPLE_TYPE $INDEX_NAME $INPUT_FILE_PATH` ?

Thank you

**Reply by hanars** [reply]
No you need to do it now before using the "Edit Datasets" functionality

**Reply by sophienguyen01** [reply]
Should I go to 'Bulk Edit Familes' or 'Bulk Edit Individuals' in 'Edit Families & Individuals'.

I only have one sample in my vcf, let say the sample column in vcf is NA12878. 
What I should fill in the tsv to submit for sample NA12878?
Family ID	Individual ID	Paternal ID	Maternal ID	Sex	Affected Status	Notes

THank you

**Reply by hanars** [reply]
Bulk Edit Individuals. Download the demo file that is available in the UI and fill out the table accordingly

**Reply by sophienguyen01** [reply]
I only have one sample in my vcf, let say the sample column in vcf is NA12878.

Here is the header in the demo file:
Family ID	Display Name	Description	Phenotype Description	MONDO ID

Should the 'FamilyID' be NA12878?

**Reply by hanars** [reply]
This is entirely up to you, this is not a question the seqr team can help with. You should be naming things in whatever way will be helpful to your group and your analysis. The upload form has a very detailed user interface that explains what each column does, what it means, and if it is required or optional. You should read that description and create the file in a way that is accurate and useful to your seqr project
